### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by dipen2000**
I created this login - sign up page for the e-commerce project so please review it.

### CODE REVIEW

File: pages/wishlist-management/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                        <div class="card-desc-product-info">
+                            <h3 class="card-desc-brand">NIKE</h3>
+                            <p class="card-desc-product-name txt-md">Nike runner</p>
+                        </div>
+                        <div class="card-desc-price-container">
+                            <h4 class="card-desc-price">Rs. 5996</h4>
+                        </div>
+                    </div>
+                    <div class="card-reactions">
+                        <img class="add-to-cart" src="https://api.iconify.design/jam:shopping-cart.svg" alt="add to cart"/>
```
Comment (m26unkwn) [inline pages/wishlist-management/wishlist.html:108]: You can add `add to cart` button.

**Review by m26unkwn — APPROVED** [review]
Otherwise code is structure is good.

File: pages/login-signup/login-signup.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    gap : 1rem;
+    width : 30rem;
+}
+
+.login-info{
+    padding : 0.8rem;
+    border : 0;
+    outline : 0;
+    border-radius: 1rem;
+}
```
Comment (ShrutiLakhani) [inline pages/login-signup/login-signup.css:27]: Clean code. LGTM!

**Review by ShrutiLakhani — APPROVED** [review]

File: CSS/reset.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    background-color: var(--background-color);
+}
+
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
```
Comment (ashishkatiyar2404) [inline CSS/reset.css:23]: try to use class for styling always don't use tag.

File: CSS/styles.css [hunk 4]
```diff
@@ -1,2 +1,207 @@
...
+
+.mb-2{
+    margin-bottom: 2rem;
+}
+
+.mt-2{
+    margin-top: 2rem;
+}
+
+.disp-block{
```
Comment (ashishkatiyar2404) [inline CSS/styles.css:43]: you should give it a meaningful name

File: CSS/styles.css [hunk 5]
```diff
@@ -1,2 +1,207 @@
...
+.highlight{
+    background-color : var(--light-test-color);
+    /* color : white; */
+    padding : 0.5rem;
+    border-radius: 5px;
+}
+
+
+.container{
+    /* border : 3px solid red;  */
```
Comment (ashishkatiyar2404) [inline CSS/styles.css]: you can remove comments

File: pages/product/product.css [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+.label{
+    font-size : 2rem;
+    font-family: var(--nav-list-items-font);
+}
+
+.product-listing-container{
+    /* border : 3px solid red; */
+    padding : 1.5rem;
+    background-color: rgba(128, 128, 128, 0.288);
```
Comment (ashishkatiyar2404) [inline pages/product/product.css]: use variable

**Review by ashishkatiyar2404 — APPROVED** [review]
your code looks good to me, also readable, you can merge after some changes💯

**Pull request merged**